src/commands/scanner/rule/list.ts

```diff
@@ -91,9 +91,9 @@ export default class List extends ScannerCommand {
 		// Map rule fields to the matching column
 		const transformedRule: Record<string, string|string[]> = {};
 		transformedRule[columns[0]] = rule.name;
-		transformedRule[columns[1]] = rule.languages;
-		transformedRule[columns[2]] = rule.categories;
-		transformedRule[columns[3]] = rule.rulesets;
+		transformedRule[columns[1]] = rule.languages.join(',');
+		transformedRule[columns[2]] = rule.categories.join(',');
+		transformedRule[columns[3]] = rule.rulesets.join(',');
```

Comment:
Wonder if this has an impact while using `--json`?

Reply:
I'm not sure. I can check and let you know.

Reply:
Doesn't look like there is. Which makes sense, because this seems to be a property of `ux.table()` rather than the JSON that's returned for the `--json` flag.